x-pack/legacy/plugins/apm/public/hooks/useTransactionBreakdown.ts

```diff
@@ -30,7 +30,7 @@ export function useTransactionBreakdown() {
         uiFilters
       });
     }
-  }, [serviceName, start, end, uiFilters]);
+  }, [serviceName, start, end, transactionType, transactionName, uiFilters]);
```

Comment:
Did you almost forget to fix what brought you here in the first place :p

Reply:
Super nice with this custom hooks linter 👍 👍

Reply:
Yeah I actually left it open because I wanted to test it until I found out that the transaction type select was broken. And then I forgot about it 😅

Reply:
But the linter saved us! Already proving its worth! :D